src/api-reference/budget/budget-category.md

```diff
@@ -3,184 +3,554 @@ title: Budget Category
 layout: reference
 ---
 
-PRE-RELEASE: SAP Concur’s Budget API is currently in PRE-RELEASE and is subject to change. An Early Adopter program is available. To become an early adopter, contact your SAP Concur Representative and have them contact Team Adaptation.
-
 # Menu
+* [Getting Started](./getting-started.html)
+* [Fiscal Year](/api-reference/budget/fiscal-year.html)
+* [Budget Category](/api-reference/budget/budget-category.html)
+* [Budget Item](/api-reference/budget/budget-header.html)
+* [Budget Tracking Field](/api-reference/budget/budget-tracking.html)
+* [Budget Adjustments](/api-reference/budget/budget-adjustments.html)
+
+# Budget Category
```

Comment:
Remove -- it's redundant with the Title in the Jekyll front matter.

Reply:
Removed